src/components/shared/Svg.css

```diff
@@ -115,3 +115,14 @@ html .arrow.expanded svg {
 .theme-dark .webpack {
   opacity: 0.5;
 }
+
+.managed-tree .tree-node .angular svg {
```

Comment:
Any reason this isn't just `.angular`, I understand the difference but I'm curious if this is safer / more maintainable / something else?

Reply:
Sorry, I only mentioned this in the issue https://github.com/devtools-html/debugger.html/issues/4733.
The icon is already used in the call stack view, thats why I added specific styling for the source pane